src/servers/src/line_writer.rs

```diff
@@ -153,6 +153,7 @@ impl LineWriter {
             schema_name: self.db,
             table_name: self.table_name,
             columns_values,
+            region_number: 0, // TODO(hl): Check if assign 0 region is ok?
```

Comment:
It depends on whether we want to create the table automatically with multi regions. This problem is also present in prometheus etc.